shuup/xtheme/models.py

```diff
@@ -181,40 +182,49 @@ def __str__(self):
 
 @python_2_unicode_compatible
 class Font(models.Model):
-    shop = models.ForeignKey(
-        "shuup.Shop", verbose_name=_("Shop"), related_name="admin_fonts", on_delete=models.CASCADE)
+    shop = models.ForeignKey("shuup.Shop", verbose_name=_("Shop"), related_name="admin_fonts", on_delete=models.CASCADE)
     name = models.CharField(max_length=128, verbose_name=_("name"), help_text=_("Font family name"))
-    css_value = models.CharField(max_length=128, verbose_name=_("Font Family CSS Property"), help_text=_("(Optional) Input css font family property directly. Ex: font-family: Roboto;"))
+    css_value = models.CharField(
+        max_length=128,
+        blank=True,
+        verbose_name=_("Font Family CSS Property"),
+        help_text=_("(Optional) Input css font family property directly. Ex: font-family: Roboto;"),
+    )
 
     woff = FilerFileField(
         verbose_name=_("WOFF Font"),
-        blank=True, null=True,
+        blank=True,
+        null=True,
         on_delete=models.SET_NULL,
-        related_name="typography_woff_fonts"
+        related_name="typography_woff_fonts",
```

Comment:
Please change `typography_woff_fonts` to  `front_woff_fonts` because it clashes with the typography model.

Reply:
Some for the fields `woff2` , `ttf`,`svg`,`eot`.

Reply:
changed to xtheme_woff_fonts as we will depricate typography and replace with this eventually